queries.sql

```diff
@@ -1,5 +1,7 @@
 /*Queries that provide answers to the questions from all projects.*/
 
+--  project0
+
 select * from animals where name like '%mon';
 
```

Comment:
- [ ] Kindly make sure to address this 👇🏻 
- Inside a transaction update the `animals` table by setting the `species` column to `unspecified`. Verify that change was made. Then roll back the change and verify that the `species` columns went back to the state before the transaction.